pymc3/distributions/multivariate.py

```diff
@@ -954,42 +960,145 @@ def _lkj_normalizing_constant(eta, n):
     return result
 
 
+class _LKJCholeskyCovRV(RandomVariable):
+    name = "_lkjcholeskycov"
+    ndim_supp = 0
+    ndims_params = [0, 0, 0]
+    dtype = "floatX"
+    _print_name = ("_LKJCholeskyCov", "\\operatorname{_LKJCholeskyCov}")
+
+    @classmethod
+    def rng_fn(cls, rng, eta, n, sd_dist, size=1):
```

Comment:
Why is it that the `sd_dist` variable here becomes a `np.ndarray` over here? Even though it's an `rv.out` when passed from `.dist()`. Is `rv.out` somehow it being evaluated?

Note: Specifically this happened when I was testing the test for `LKJCholeskyCov` from: https://github.com/pymc-devs/pymc3/blob/7be2025c4091e450f9e9dd050eb0dca08995c83f/pymc3/tests/test_distributions_random.py#L2187-L2197 and the `rng_fn` was called from `Model.set_initval`